Example/Database/Tests/Unit/FLevelDBStorageEngineTests.m

```diff
@@ -532,8 +532,12 @@ - (void)testIntegersAreReturnedsAsIntegers {
   XCTAssertEqual(CFNumberGetType((CFNumberRef)actualInt), kCFNumberSInt64Type);
   XCTAssertEqualObjects([actualLong stringValue], [longValue stringValue]);
   XCTAssertEqual(CFNumberGetType((CFNumberRef)actualLong), kCFNumberSInt64Type);
-#if TARGET_OS_MACCATALYST
-  // Catalyst uses int128_t but CFNumber still calls it 64 bits
+#ifdef TARGET_OS_IOS
+  // Catalyst and iOS 13 use int128_t but CFNumber still calls it 64 bits.
+  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion >= 13) {
```

Comment:
I think the check in the `#else` block wouldn't run on iOS 12 and below anymore,  since now this is running for iOS.

Reply:
Yep. Fixed. Thanks.